service/api_patch.go

```diff
@@ -81,10 +80,6 @@ func (as *APIServer) submitPatch(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(data.BackportInfo.SHA) == 0 && len(data.BackportInfo.PatchID) == 0 {
-		data.BackportInfo.PatchID = data.BackportOf
```

Comment:
Maybe we should keep this if statement and return an error to the user that their CLI might be out of date? I'm concerned otherwise that some user somewhere hasn't updated and then they'll see unintended unhelpful behavior.

Reply:
that's okay with me as an intermediate step. Will you be okay with me removing it at a later time?

Reply:
Yeah I suppose we have to eventually :) but considering this check is made regardless of whether the user is trying to backport, this will ensure users who use the CLI at least like once per month will see it haha